Conceptpower+Spring/src/main/java/edu/asu/conceptpower/rest/ConceptIDLookup.java

```diff
@@ -91,7 +91,7 @@
 
         IConceptMessage msg = messageFactory.getMessageFactory(acceptHeader).createConceptMessage();
 
-        if (entry != null) {
+        if (entry != null && !entry.isDeleted()) {
```

Comment:
Ok, so we seem to go back and forth on this (and I think I was confused at some point too), but if I call .../Concept?id=id_of_deleted_concept, I still want to get the information about the concept I was asking for with the note that the concept was deleted. This happened in the old system, e.g.:
https://diging-dev.asu.edu/conceptpower-latest/rest/Concept?id=CON57c7c59b-a160-4bc1-8f4e-f9a72e14dce5

If Conceptpower suddenly stops returning results for deleted Concepts that might break other applications relying on always getting a response. When a deleted concept was actually wrapped or merged, then it's debatable what should be returned, but it should always return something.